src/kvstore/Listener.cpp

```diff
@@ -251,5 +252,17 @@ std::pair<int64_t, int64_t> Listener::commitSnapshot(const std::vector<std::stri
     return std::make_pair(count, size);
 }
 
+void Listener::reset() {
+    std::lock_guard<std::mutex> guard(raftLock_);
+    committedLogId_ = proposedTerm_ = lastLogTerm_ = term_ = lastApplyLogId_ = 0;
+    lastTerm_ = lastId_ = -1;
+    wal_->reset();
+    persist(lastApplyLogId_, lastTerm_, lastApplyLogId_);
+}
```

Comment:
@critical27 please help me to take a look at this method, I am not sure whether it is reasonable. thanks.

Reply:
Let me modify the listener, I think could simplify it, the `lastTerm_` and `lastId_` is useless.